lib/foreman/logging.rb

```diff
@@ -259,7 +259,15 @@ def indent_lines(string)
 
     class MultilineRequestPatternLayout < MultilinePatternLayout
       def indent_lines(string)
-        string.gsub("\n", "\n #{::Logging.mdc['request'].split('-').first} | ")
+        if request_id
```

Comment:
Will this only ever be `nil` or the actual request_id ?

Reply:
It appears so!